jwql/instrument_monitors/common_monitors/dark_monitor.py

```diff
@@ -706,7 +705,7 @@ def process(self, file_list):
         step_args = {'dark_current': {'skip': True}}
 
         # Call the pipeline
-        outputs = run_parallel_pipeline(pipeline_files, "dark", ["rate"], self.instrument, step_args=step_args)
+        _ = run_parallel_pipeline(pipeline_files, "dark", ["rate"], self.instrument, step_args=step_args)
```

Comment:
you don't need to save the return if you're not going to use the returned file list.  You can get rid of the `_ =`